main/src/ca/uwaterloo/flix/language/phase/Parser.scala

```diff
@@ -1836,21 +1812,21 @@ class Parser(val source: Source) extends org.parboiled2.Parser {
 
     def Class: Rule1[Name.Ident] = UpperCaseName
 
-    def QualifiedClass: Rule1[Name.QName] = UpperCaseQName
+    def QualifiedClass: Rule1[Name.QName] = QName
```

Comment:
Should we just delete these different names then?

Reply:
This is the only change we should potentially consider for this PR.

Reply:
Maybe. I wasn't sure whether it was best to keep things in the parser or not.

Reply:
Can we remove everything and just have QName? Why not?

Reply:
We can. It just seemed that the style was to separate everything :)